{
  Mockito.verify(mock,0).clear();
  try {
    Mockito.verify(mock,15).clear();
    fail();
  }
 catch (  NumberOfInvocationsError e) {
    assertTrue(e.getMessage().endsWith("Expected 15 times but was 0"));
  }
}
